ambry-tools/src/main/java/com.github.ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -124,12 +124,12 @@ static void createCluster(String destZkString, String destClusterName) {
         DedicatedZkClientFactory.getInstance().buildZkClient(new HelixZkClient.ZkConnectionConfig(destZkString));
     destZkClient.setZkSerializer(new ZNRecordSerializer());
     HelixAdmin destAdmin = new ZKHelixAdmin(destZkClient);
-    if (destAdmin.getClusters().contains(destClusterName)) {
-      System.err.println("Failed to create cluster becuase " + destClusterName + " already exist.");
-      return;
-    }
-    ClusterSetup clusterSetup = new ClusterSetup(destZkString);
-    clusterSetup.addCluster(destClusterName, true);
+//    if (destAdmin.getClusters().contains(destClusterName)) {
```

Comment:
Not good to have commented code.  Is it still needed?

Reply:
Removed `//` .